### PR TITLE
Add support for separate projection basis

### PR DESCRIPTION
**Pull request by akva2**
Admittedly this is kind of not needed, but I need something to test the LR support with :)

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this ifem=593 please

**Comment by akva2** [comment]
jenkins build this ifem=593 please

**Comment by akva2** [comment]
jenkins build this please

File: Poisson.C [hunk 1]
```diff
@@ -375,6 +376,9 @@ PoissonNorm::PoissonNorm (Poisson& p, int integrandType, VecFunc* a) :
 }
 
 
+PoissonNorm::~PoissonNorm() = default;
+
+
```
Comment (kmokstad) [inline Poisson.C:381]: Is there any practical difference between this and an empty destructor? And if yes, why not keep the default declaration in the header file.

**Review by kmokstad — APPROVED** [review]
Looks fine to me

File: Poisson.C (hunk 1 above)
Comment (akva2) [reply Poisson.C:381]: not as such, but the big difference is that it's the dtor is generated in the source file. in the source file we know the definition of the fields type. hence we avoid pulling in the fields header in the header.

**Pull request merged**